src/app/page.tsx

```diff
@@ -1,9 +1,137 @@
-import styles from './page.module.css';
+'use client';
+
+import COLORS from '@/styles/colors';
+import { H5, H3 } from '../styles/text';
+import Button from '../components/Button';
+import image from '../../public/HomePage.webp';
+import {
+  TitleSection,
+  PageContainer,
+  ButtonDiv,
+  StatisticsSection,
+  StatNumber,
+  StatLabel,
+  StatContainer,
+  HowYouCanHelpContainer,
+  Title,
+  Subtitle,
+  ServiceContainer,
+  ServiceIcon,
+  ServiceTitle,
+  ServiceDescription,
+  ServicesDiv,
+  MissionValuesContainer,
+  MissionStatement,
+} from './styles';
 
 export default function Home() {
   return (
-    <main className={styles.main}>
-      <div className={styles.description}>immigration justice project</div>
-    </main>
+    <PageContainer>
+      <TitleSection>
+        <H5>IMMIGRATION JUSTICE PROJECT</H5>
+        <H3 $color="white" $fontWeight="400">
+          The Immigration Justice Project (IJP) seeks to promote due process and
+          access to justice through the provision of high quality legal services
+          on behalf of indigent immigrants and asylum seekers.
+        </H3>
+        <ButtonDiv>
+          <Button
+            $primaryColor={COLORS.goldMid}
+            $secondaryColor={COLORS.blueDarker}
```

Comment:
@kyrenetam on hover for gold buttons, do we want the color of the button to change to a darker gold perhaps?